changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "3.1.0": [
+      "[Added] You can diff a consecutive range of commits - #14682",
+      "[Added] Improve submodule support with a new diff screen with more detailed information - #15176",
```

```suggestion
      "[Added] Submodule changes now display an explanatory, interactive diff screen  - #15176",
```
Just my 2 cents: "Improve submodule support" feels a bit lack luster for this pretty significant addition for usability. :) Tho, not sure if my suggestion is the best way to say it.. so feel free to take or leave it.  Other thoughts were "now display a more intuitive, interactive diff screen"

---

I like it!! Thank you ❤️